server/account/src/utils.ts

```diff
@@ -187,3 +187,7 @@ export function areDbIdsEqual (obj1: any, obj2: any): boolean {
 
   return obj1 === obj2
 }
+
+export function isShallowEqual (obj1: Record<string, any>, obj2: Record<string, any>): boolean {
+  return Object.keys(obj1).every((k) => obj1[k] === obj2[k]) && Object.keys(obj1).length === Object.keys(obj2).length
```

Comment:
nit: swap conditions for better performance?

Reply:
Thanks, updated